tests/integration/cairo1_compiler_bindings/library_functions_test.py

```diff
@@ -151,19 +215,42 @@ def _args_validator(test_case_name: str, *args: Any, **kwargs: Any):
 
 def test_prepare(datadir: Path):
     expected_calldatas = {
-        "test_prepare": [101, 202, 303, 405, 508, 613, 721],
+        "test_prepare": [101, 202, 613, 721],
         "test_prepare_tp": [3, 2, 1],
         "test_prepare_no_args": [],
     }
+    return_values = {
```

Comment:
I know but I consider this as better looking, you have this structure now that you have all the "configs" in the beginning, then the callbacks that use these configs, and then a final invocation. I'd stay with what is but if you have a strong opinion I don't mind moving it.

Reply:
Ok, I will resolve this since you said it's a nitpick and "not a deal-breaker".
If you feel like this should be addressed, feel free to open an issue or something, thanks!